dsgrid/utils/dataset.py

```diff
@@ -165,7 +165,9 @@ def check_null_value_in_unique_dimension_rows(dim_table, exclude_columns=None):
 def is_noop_mapping(records: pyspark.sql.DataFrame) -> bool:
     """Return True if the mapping is a no-op."""
     return records.filter(
-        (records.from_id != records.to_id) | (records.from_fraction != 1.0)
+        (records.to_id.isNull() & ~records.from_id.isNull())
```

Comment:
Should there also be:

```
(~records.to_id.isNull() & records.from_id.isNull())
```

?

Reply:
Good point. Null on the from_id side probably doesn't happen? That would correspond to being explicit about "this thing exists in the project but we aren't mapping anything to it."

Reply:
Sounds good. It would be annoying bug to figure out and the extra check won't harm anything.